### PR TITLE
Change search placeholder

### LINKED ISSUE #937: Sales page search field has wrong prefilled text
On Sales page, user searches for "Buyer".
On Purchases page, user searches for "Vendor".
On Cases page, user searches for "Vendor/Buyer"? (I have no case, need to check this).

salesvendor

### PR DESCRIPTION
**Pull request by jjeffryes**
Changes the transaction search placeholder text so it applies to all 3 transaction tabs. 

Closes #937

### CODE REVIEW

**Comment by rmisio** [comment]
@morebrownies Are you find with using the generic `Person ID` here?

I know you had mentioned you wanted this:

```
The placeholder is correct when you're on the Purchases tab, but incorrect when on the Sales tab.

We'll probably want to change the placeholder text to Search by Order #, Title or Buyer when on the Sales tab.
```

**Comment by jjeffryes** [comment]
It's different for each tab now.

**Comment by jjeffryes** [comment]
@morebrownies this was updated to the less lazy solution. :)

**Comment by rmisio** [comment]
> this was updated to the less lazy solution. :)

Oh... no `Person ID` anymore? :(

File: js/languages/en_US.json [hunk 1]
```diff
@@ -1279,7 +1279,9 @@
       "dateAsc": "Newest",
       "dateDesc": "Oldest"
     },
-    "placeholderSearch": "Search by Order #, Title or Vendor",
+    "placeholderSearchsales": "Search by Order #, Title or Buyer",
```
Comment (rmisio) [inline js/languages/en_US.json]: It's minor, but can we use proper camel-case here and then you could just use the `capitalize` utility function in `tab.html`?

Comment (jjeffryes) [reply js/languages/en_US.json]: Will do.

**Pull request merged**